app/helpers/application_helper.rb

```diff
@@ -10,4 +12,36 @@ def user_status(flag)
   def url_for_client
     current_user.admin? ? admin_clients_url : clients_url
   end
+
+  def url_for_project
+    current_user.admin? ? admin_projects_url : projects_url
+  end
+
+  def url_for_root
+    current_user.admin? ? admin_projects_url : root_url
+  end
+<<<<<<< HEAD
```

Comment:
?

Reply:
if you don't know how to resolve these, discuss with me or any other senior developer.